src/expandableCalendar/index.tsx

```diff
@@ -581,15 +583,15 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
     <View testID={testID} style={containerStyle}>
       {screenReaderEnabled ? (
         <Calendar
-          testID="calendar"
+          testID={`${testID}.calendarAccessible`}
           {...others}
           theme={themeObject}
           onDayPress={_onDayPress}
           hideExtraDays
           renderArrow={_renderArrow}
         />
       ) : (
-        <Animated.View ref={wrapper} style={wrapperStyle} {...panResponder.panHandlers}>
+        <Animated.View testID={`${testID}.expandableHeader`} ref={wrapper} style={wrapperStyle} {...panResponder.panHandlers}>
```

Comment:
This is not a header... This is the "expandable" calendar/wrapper

Reply:
The header for the vertical state is the animated header in `renderAnimatedHeader()`. Maybe worth adding a testID there as well...